prior:
  # imu 0
  # gyroscope_bias: [-0.546656, -6.59877e-05, 0.00244956]
  # accelerometer_bias: [0.0737028, -0.0178264, -0.000426039]
  
  # imu 1
  gyroscope_bias: [0.00104398, 0.000137212, -0.000153058]   
  accelerometer_bias: [0.00286941, -0.0323794, -0.00220881]
  
  